test/param/number-list.spec.js

```diff
@@ -41,4 +41,54 @@ describe('Testing NumberList Parameter', () => {
       }
     })).to.throw('Invalid Value for json-Parameter "list" provided.');
   });
+
+  describe('Testing optional parameter "minItemValue"', () => {
+    let queryParamInValidMin;
+    let bodyParamInValidMin;
+    before(() => {
+      queryParamInValidMin = api.NumberList('list', 'query', { minItemValue: 1 });
+      bodyParamInValidMin = api.NumberList('list', 'json', { minItemValue: 1 });
+    });
+
+    it('Testing invalid min-value of list item in query param', () => {
+      expect(() => queryParamInValidMin.get({
+        queryStringParameters: {
+          list: '[0, 1, 200]'
+        }
+      })).to.throw('Invalid Value for query-Parameter "list" provided.');
+    });
+
+    it('Testing invalid min-value of list item in json param', () => {
+      expect(() => bodyParamInValidMin.get({
+        body: {
+          list: [0, 1, 200]
+        }
+      })).to.throw('Invalid Value for json-Parameter "list" provided.');
+    });
+  });
+
+  describe('Testing optional parameter "maxItemValue"', () => {
+    let queryParamInValidMax;
+    let bodyParamInValidMax;
+    before(() => {
+      queryParamInValidMax = api.NumberList('list', 'query', { maxItemValue: 100 });
+      bodyParamInValidMax = api.NumberList('list', 'json', { maxItemValue: 100 });
+    });
+
+    it('Testing invalid max-value of list item in query param', () => {
+      expect(() => queryParamInValidMax.get({
+        queryStringParameters: {
+          list: [101, 0, 1]
```

Comment:
Needs to be a string value:
```suggestion
          list: '[101, 0, 1]'
```